File: SDKs/amazonaws.com/autoscaling/2011-01-01/python/src/sdk/models/shared/enabledmetric.py
from dataclasses import dataclass, field
from typing import Optional


@dataclass
class EnabledMetric:
    r"""EnabledMetric
    Describes an enabled metric.
    """
    
    granularity: Optional[str] = field(default=None)
    metric: Optional[str] = field(default=None)
    

File: SDKs/amazonaws.com/autoscaling/2011-01-01/typescript/src/sdk/models/shared/enabledmetric.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// EnabledMetric
/** 
 * Describes an enabled metric.
**/
export class EnabledMetric extends SpeakeasyBase {
  @SpeakeasyMetadata()
  granularity?: string;

  @SpeakeasyMetadata()
  metric?: string;
}
